test/models/domain/domain_version_test.rb

```diff
@@ -13,30 +13,30 @@ def teardown
     super
   end
 
-  def test_assigns_creator_to_paper_trail_whodunnit
-    duplicate_domain = prepare_duplicate_domain
-
-    PaperTrail.whodunnit = @user.id_role_username
-    assert_difference 'duplicate_domain.versions.count', 1 do
-      duplicate_domain.save!
-    end
-
-    assert_equal(duplicate_domain.creator, @user)
-    assert_equal(duplicate_domain.updator, @user)
-    assert_equal(duplicate_domain.creator_str, @user.id_role_username)
-    assert_equal(duplicate_domain.updator_str, @user.id_role_username)
-  end
-
-  def test_assigns_updator_to_paper_trail_whodunnit
-    PaperTrail.whodunnit = @user.id_role_username
-
-    assert_difference '@domain.versions.count', 1 do
-      @domain.apply_registry_lock
-    end
-
-    assert_equal(@domain.updator, @user)
-    assert_equal(@domain.updator_str, @user.id_role_username)
-  end
+  # def test_assigns_creator_to_paper_trail_whodunnit
+  #   duplicate_domain = prepare_duplicate_domain
+  #
+  #   PaperTrail.whodunnit = @user.id_role_username
+  #   assert_difference 'duplicate_domain.versions.count', 1 do
+  #     duplicate_domain.save!
+  #   end
+  #
+  #   assert_equal(duplicate_domain.creator, @user)
+  #   assert_equal(duplicate_domain.updator, @user)
+  #   assert_equal(duplicate_domain.creator_str, @user.id_role_username)
+  #   assert_equal(duplicate_domain.updator_str, @user.id_role_username)
+  # end
+  #
+  # def test_assigns_updator_to_paper_trail_whodunnit
+  #   PaperTrail.whodunnit = @user.id_role_username
+  #
+  #   assert_difference '@domain.versions.count', 1 do
+  #     @domain.apply_registry_lock
+  #   end
+  #
+  #   assert_equal(@domain.updator, @user)
+  #   assert_equal(@domain.updator_str, @user.id_role_username)
+  # end
```

Comment:
Is it WIP? If not, I guess we're better off with simply deleting this file as nothing is being tested.

Reply:
Yep, that's a forgotten leftover. Thanks.